.spyproject/config/codestyle.ini

```diff
@@ -0,0 +1,8 @@
+[codestyle]
```

Comment:
I see these files were added to the `.gitignore` but are still present. Were they accidentally included? I think they should probably be removed since not everyone will necessarily use this IDE or have the same configuration.

Reply:
Yes this was accidental and the files can be removed